.scripts/build_steps.sh

```diff
@@ -20,6 +20,7 @@ conda-build:
 CONDARC
 
 conda install --yes --quiet conda-forge-ci-setup=3 conda-build pip -c conda-forge
+pip instal git+https://github.com/isuruf/conda-build@patch-11
```

Comment:
`install`?

Reply:
This line should not have been committed